### ISSUE TITLE
Reminders for Todo List tasks

### ISSUE THREAD

**Issue by LunarFang416** [encorelab/ck-board#349]
Use the existing notificationService to generate notifications to remind students of their pending or overdue tasks. Send one notification the day before, and one notification the day a task is due. Also send a notification for an overdue task as well.

**Comment by JoelWiebe** [comment]
Are there some ideas you could get from here: https://stackoverflow.com/questions/54002431/how-to-trigger-events-in-node-js-when-a-future-date-time-stored-in-database-is

**Comment by LunarFang416** [comment]
Yeah thats a good Idea. Will work on this, this week. I think it will be easier to implement a listener in the backend which checks for any pending notifications after certain intervals.
